ML_2011_LU_eusilc_cs.do

```diff
@@ -27,11 +27,11 @@ replace ml_dur2 = 8 		if country == "LU" & year == 2011 & ml_eli == 1
 
 * BENEFIT (monthly)
 /*	-> 100% earning
-	-> maximum:  €9614.82  gross/month 		*/
+	-> maximum:  €100,350  gross/month 		*/

```

Comment:
I made a stupid mistake here, needs to be divided by twelve since it's €100,350/YEAR